examples/deployment/README.md

```diff
@@ -0,0 +1,731 @@
+--set prometheusoperator.prometheusOperator.createCustomResource=false \
+--set installEFK=true \
+--namespace coherence-example-ns \
+--set imagePullSecrets[0].name=coherence-example-secret \
+--name coherence-operator
+```    
+
+Confirm the operator and all associated pods are running:
+
+```bash
+kubectl get pods -n coherence-example-ns
+
+NAME                                                     READY   STATUS    RESTARTS   AGE
+coherence-operator-65bc5c7b49-9l5br                      1/1     Running   0          80s
+coherence-operator-grafana-8454698bcf-5dqvm              2/2     Running   0          80s
+coherence-operator-kube-state-metrics-6dc8675d87-jz4d7   1/1     Running   0          80s
+coherence-operator-prometh-operator-58d94ffbb8-lhgsn     1/1     Running   0          80s
+coherence-operator-prometheus-node-exporter-lxtvs        1/1     Running   0          80s
+elasticsearch-f978d6fdd-69nfd                            1/1     Running   0          80s
+kibana-9964496fd-rbslk                                   1/1     Running   0          80s
+prometheus-coherence-operator-prometh-prometheus-0       3/3     Running   1          66s
+```
+      
+## Port Forward Grafana and Kibana ports
+
+If you have enabled metrics or ELK integration you can port-forward the ports for these components using the scripts
+in the `examples/bin/` directory.
+
+*   Port-forward Grafana for viewing metrics
+  
+    ```bash
+    ./port-forward-grafana.sh coherence-example-ns   
+
+    Port-forwarding coherence-operator-grafana-8454698bcf-5dqvm in coherence-example-ns
+    Open the following URL: http://127.0.0.1:3000/d/coh-main/coherence-dashboard-main
+    Forwarding from 127.0.0.1:3000 -> 3000
+    Forwarding from [::1]:3000 -> 3000
+    ``` 
+      
+*   Port-forward Kibana for viewing log messages
+   
+    ```bash
+    ./port-forward-kibana.sh coherence-example-ns       
+  
+    Port-forwarding kibana-67c4f74ffb-nspwz in coherence-example-ns
+    Open the following URL: http://127.0.0.1:5601/
+    Forwarding from 127.0.0.1:5601 -> 5601
+    Forwarding from [::1]:5601 -> 5601
+    ```            
+
+# Run the Examples
+
+Change to the `examples/deployment` directory to run the following commands.
+
+## Example 1 - Coherence cluster only
+
+The first example uses the yaml file [src/main/yaml/example-cluster.yaml](src/main/yaml/example-cluster.yaml) which
+defines a single role `storage` which will store cluster data. The configuration under the main `spec` entry will be the defaults for all
+roles unless they are overridden by a specific role.
+
+This saves duplication of role configuration leading to less configuration errors.
+
+1.  Install the Coherence cluster `storage` role
+
+    ```bash   
+    kubectl -n coherence-example-ns create -f src/main/yaml/example-cluster.yaml 
+    ```       
+
+    List the created `coherenceclusters` and `coherenceroles` using:
+
+    ```bash
+    kubectl get coherenceclusters -n coherence-example-ns
```

Comment:
You can actually do `kubectl -n coherence-example-ns get coherence` and that will display all `CoherenceCluster` _and_ `CoherenceRole` resources

Reply:
ok cool.